dspace-api/src/main/java/org/dspace/content/dao/impl/BitstreamDAOImpl.java

```diff
@@ -158,9 +170,9 @@ public int countWithNoPolicy(Context context) throws SQLException {
 
     @Override
     public Iterator<Bitstream> findAll(Context context, int limit, int offset) throws SQLException {
-        Query query = createQuery(context, "select b FROM Bitstream b");
-        query.setFirstResult(offset);
-        query.setMaxResults(limit);
-        return iterate(query);
+        Map<String, Object> map = new HashMap<>();
+//        map.put("sizeBytes", 12);
```

Comment:
What is this commented out line doing here? Is there a reason for it?

Reply:
I used this to test the method's functionality and didn't remove it, there is no other reason for it and will be removed